h2o-core/src/main/java/water/api/ParseHandler.java

```diff
@@ -26,20 +22,32 @@ public ParseV3 parse(int version, ParseV3 parse) {
                                       parse.domains, parse.na_strings,
                                       null,
                                       new ParseWriter.ParseErr[0], parse.chunk_size,
-                                      parse.decrypt_tool != null ? parse.decrypt_tool.key() : null);
+                                      parse.decrypt_tool != null ? parse.decrypt_tool.key() : null, parse.skipped_columns);
 
     if (parse.source_frames == null) throw new H2OIllegalArgumentException("Data for Frame '" + parse.destination_frame.name + "' is not available. Please check that the path is valid (for all H2O nodes).'");
     Key[] srcs = new Key[parse.source_frames.length];
     for (int i = 0; i < parse.source_frames.length; i++)
       srcs[i] = parse.source_frames[i].key();
 
+    if ((setup.getParseType().name().toLowerCase().equals("svmlight") ||
+            (setup.getParseType().name().toLowerCase().equals("avro") ))
+            && ((setup.getSkippedColumns() != null) && (setup.getSkippedColumns().length >0)))
+      throw new H2OIllegalArgumentException("Parser: skipped_columns are not supported for SVMlight or Avro parsers.");
+
+    if ((setup.getSkippedColumns() !=null &&
```

Comment:
I don't understand this condition. 

We should probably subtract `nSkipped` from `nParsed`, where `nSkipped` is number of skipped cols and `nParsed` is number of columns parsed from the dataset. If the result is 0 (assert for it never to be negative ?), then the exception is thrown ? Plus the null checks of course.

Am I missing something ?